crescent/internal/app_command.py

```diff
@@ -146,7 +146,7 @@ class AppCommandMeta:
     owner: Any
     """The function or class that was used to create the command"""
     callback: CommandCallbackT
-    autocomplete: dict[str, TransformedAutocompleteCallbackT] = field(default_factory=dict)
+    autocomplete: dict[str, TransformedAutocompleteCallbackT[Any]] = field(default_factory=dict)
```

Comment:
Could you type this as `int | str | float` in whatever order instead of `Any`? I think it would be more accurate.

Reply:
No, it's `TypeVar(..., int, str, float)`, which won't accept Union[int, str, float] - only one of them.

Reply:
Ah, could you use the type var from `types.py` then?

Reply:
![35808](https://github.com/hikari-crescent/hikari-crescent/assets/55721663/7d0da29d-da98-49f3-8384-dcf23471c4e4)

Reply:
Adding it as a generic would just cause more problems and be pretty useless I think

Reply:
alr `Any` is fine then